src/gax-internal/tests/grpc_auth.rs

```diff
@@ -97,19 +97,13 @@ mod test {
             .await?;
 
         let result = send_request(client, "auth fail").await;
-        assert!(result.is_err());
-
-        if let Err(e) = result {
-            if let Some(cred_err) = e.as_inner::<CredentialsError>() {
-                assert!(
-                    cred_err.is_transient(),
-                    "Expected a retryable CredentialsError, but got non-retryable"
-                );
-            } else {
-                panic!("Expected a CredentialsError, but got some other error: {e:?}");
-            }
-        }
-
+        use std::error::Error;
```

Comment:
nit: pop this out?

Reply:
ignore this, there are too many instances for me to care.

Reply:
Fixed all of them.